lib/src/components/_stacks-tables.less

```diff
@@ -316,4 +328,8 @@
     max-width: @su24;
     padding-right: @su2;
     padding-left: @su2;
+
+    input {
```

Comment:
This feels dirty and temporary. Think we could comment that this is an override to bad defaults in Core?

Reply:
Sure, and hopefully this will get standardized when balpha finishes his table sorting stuff and we know what markup should be in the `<th>` to make sorting happen.

Reply:
Actually... should we just advise using an `.m0` on the `input` here?

Reply:
Yeah, now that you mention it, I think that's where my head's at too.

Reply:
Ok, I'll update.